resin/models/device_os.py

```diff
@@ -24,6 +24,26 @@ def __init__(self):
         self.base_request = BaseRequest()
         self.settings = Settings()
 
+    def get_config(self, app_id):
+        """
+        Get an application config.json
+
+        Args:
+            app_id (str): application id.
+
+        Returns:
+            dict: application config.json
+
+        """
+
+        data = {
+            'appId': app_id
+        }
+        return self.base_request.request(
+            'download-config', 'POST', data=data,
+            endpoint=self.settings.get('api_endpoint')
+        )
```

Comment:
I think you almost certainly want to allow more options here too. The JS SDK adds the appId, but also whatever else you want to include: https://github.com/resin-io/resin-sdk/blob/45c0ed0cb1af4833952d84b03cbd8a8ed2d30e28/lib/models/os.coffee#L369. As is, you can't use this to configure wifi/network settings/poll intervals etc.

Reply:
Yeah, I've seen we support options in the JS SDK and I will implement it (not just for this method) in the next PR. This PR will focus on updating the SDK with latest API.

Reply:
Alright, fair enough. I don't think this method is super useful without that, but adding more options can wait, it's not a problem.